src/components/Link/Link.tsx

```diff
@@ -10,7 +10,7 @@ export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
   to?: To;
   variant?: LinkVariants;
   isNavLink?: boolean;
-  isCurrentPage?: boolean;
+  isCurrentPage?: boolean | undefined;
```

Comment:
Was it throwing any warnings/errors before adding this? the ? making it optional should automatically add an | undefined behind the scenes.